iocore/eventsystem/UnixEventProcessor.cc

```diff
@@ -74,10 +89,16 @@ class ThreadInitByFunc : public Continuation
 } Thread_Init_Func;
 }
 
+void *
+ThreadAffinityInitializer::alloc_hugepage_stack(size_t stacksize)
+{
+  return ats_hugepage_enabled() ? ats_alloc_hugepage(stacksize) : ats_memalign(ats_pagesize(), stacksize);
```

Comment:
Since you are adding this code, which might call ats_alloc_hugepage(), wouldn't there be a place where a call to ats_alloc_hugepage() is removed? I don't see that in this PR, so I must be missing something.